DevEdu.API/DevEdu.DAL/Repositories/UserRoleRepository.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Data;
+using Dapper;
+using DevEdu.DAL.Models;
+
+
+namespace DevEdu.DAL.Repositories
+{
+    public class UserRoleRepository : BaseRepository, IUserRoleRepository
```

Comment:
no need to create a separate repository for this
please move all its methods to UserRepository